app/views/admin/hire/forms/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render Admin::ContainerComponent.new do %>
+  <%= render Admin::ContentComponent.new do %>
+    <%= render Admin::DescriptionListComponent.new(t(".title")) do |list| %>
+      <% list.with_description "Full name" do %>
+        <%= render Admin::BasicLinkComponent.new(@form.business.name, business_path(@form.business)) %>
```

Comment:
Let's also add their email address in this same column that links with a `mailto:`.